Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+def config = jobConfig {
+    slackChannel = 'tools-eng'  // TODO: Change this when Viktor provides one
+}
+
+def job = {
+    if (config.isRealJob) {
+        // TODO: Update this to version per commit and publish to Artifactory once Viktor provides more details
+        stage('Package') {
+            sh 'scripts/package.sh'
+        }
+
+    } else {
+        stage('Lint') {
```

Comment:
if we're heading towards managing this build process with jenkins, might make sense to always have this as an explicit jenkinsfile stage and just remove the call in `scripts/package.sh` to the lint script. potentially nicer during development too as only having to clean up lint issues as a final step can be nicer than constantly getting lint output when you're just trying to get something functioning first.